.github/CODEOWNERS

```diff
@@ -0,0 +1,55 @@
+# Default code owners
+* @pablintino @lewisdenny
+
+# Adoption
+roles/adoption_osp_deploy @jistr @sathlan @ciecierski @frenzyfriday
+
+# Compliance
+roles/compliance @vakwetu
+
+# DCN
+roles/ci_dcn_site @fultonj @sbekkerm @krcmarik
+playbooks/dcn.yml @fultonj @sbekkerm @krcmarik
+
+# Must gather
+roles/os_must_gather @fmount
+
+# NFV
+roles/idrac_configuration @eshulman2 @rdiazcam @mnietoji
+roles/ovirt @eshulman2 @rdiazcam @mnietoji
+roles/switch_config @eshulman2 @rdiazcam @mnietoji
+roles/tofu
+
+# Ovirt
+roles/ovirt @dpinhas
```

Comment:
I believe RH employee should be code-owner of this role :)

Reply:
oh shoot :D

Reply:
It's opensource mate, get amongst it :D

Reply:
@lewisdenny Of course! Seriously I'll be happy to contribute, as long as I'm pinged/mentioned in GitHib issues.